subworkflows/nf-core/fgbio_create_umi_consensus/main.nf

```diff
@@ -0,0 +1,93 @@
+//
+// Runs FGBIO tools to remove UMI tags from FASTQ reads
+// Convert them to unmapped BAM file, map them to the reference genome,
+// use the mapped information to group UMIs and generate consensus reads
+//
+
+params.fastqtobam_options      = [:]
+params.bam2fq_options          = [args: "-T RX"]
+params.bwamem_options          = [args: "-p -C -M"]
+params.bwamem2_options          = [args: "-p -C -M"]
+params.samblaster_options      = [args: "-M --addMateTags", suffix:'_processed']
+params.groupreadsbyumi_options = [:]
+params.groupumistrategy        = "Adjacency"
+params.umiconsensus_options    = [args: '-M 1 -S Coordinate', suffix: '_umiconsensus']
+params.aligner                 = "bwa-mem"
+
+include { FGBIO_FASTQTOBAM                  as FASTQTOBAM }         from '../../../modules/fgbio/fastqtobam/main'                        addParams( options: params.fastqtobam_options )
```

Comment:
The addParams part can be removed with the new syntax as far as I understand

Reply:
I'm aware of that, but implementing it requires update of all modules involved first, doesn't it?
You need to put `ext.args = ` in a config (that's easy), but also recode the individual modules to use it in the script section. An effort worth doing, but it will definitely postpone all of this.

Reply:
now updated :)